metasyn/metaframe.py

```diff
@@ -120,46 +122,70 @@ def fit_dataframe(
         MetaFrame:
             Initialized metasyn metaframe.
         """
-        if privacy is None:
-            privacy = BasicPrivacy()
-        if spec is None:
-            spec = {}
+        if meta_config is None:
+            if privacy is None:
+                privacy = {"name": "none"}
+            elif isinstance(privacy, BasePrivacy):
+                privacy = privacy.to_dict()
+            var_specs = [] if var_specs is None else var_specs
+            dist_providers = dist_providers if dist_providers is not None else ["builtin"]
+            meta_config = MetaConfig(var_specs, dist_providers, privacy)
```

Comment:
My comments above would lead to pretty small changes here I think, and not elsewhere.

Reply:
I'm not sure which comments you are thinking about here?